### ISSUE TITLE
Ruby 2.0.0 isn't good enough

### ISSUE THREAD

**Issue by martinthomson** [cabo/kramdown-rfc2629#51]
We just spent a while debugging an issue with an apparent error with the YAML parser.  It seems like the version of ruby (2.0.0) has issues.  Installing with a more recent version of gem/ruby fixed the issue.

**Comment by elear** [comment]
Is 2.3 good enough?

**Comment by cabo** [comment]
On Oct 4, 2017, at 02:58, Martin Thomson <notifications@github.com> wrote:
> 
> We just spent a while debugging an issue with an apparent error with the YAML parser. It seems like the version of ruby (2.0.0) has issues. Installing with a more recent version of gem/ruby fixed the issue.

Oops, I missed this during my vacation.
Russ Housley alerted me to this and I pushed 1.2.7 that should fix this.

(The problem was that the safer API to parse YAML I switched to in July only works on Ruby ≥ 2.1 — I installed a workaround for 1.9/2.0 now, but of course the safer API is better, as theoretically your laptop can be taken over by someone pushing a well-crafted YAML file.  Support has ended for 2.0 in 2016.)

Grüße, Carsten
